add assets flag

timsim00 opened the pull request.

timsim00 committed:
add assets flag

yoshuawuyts commented:
Heya, thanks for the PR! Could you perhaps explain your reasoning behind this? Bankai should be checking for the `assets/` directory out of the box already!

timsim00 commented:
Hmm, I added an image and it didn't show up until I added this flag.  Maybe
I did something wrong.

<img src="/assets/logo.jpeg" alt="logo">

On Mon, Oct 30, 2017 at 7:59 AM, Yoshua Wuyts <notifications@github.com>
wrote:

> Heya, thanks for the PR! Could you perhaps explain your reasoning behind
> this? Bankai should be checking for the assets/ directory out of the box
> already!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/choojs/create-choo-app/pull/32#issuecomment-340436226>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF_1ZqzifvR0HGKIugTkJu9aOqkGZCWWks5sxcg4gaJpZM4QDmbH>
> .
>

timsim00 commented:
Closing.  This doesn't appear to be an issue anymore.
